bazel/foreign_cc/BUILD

```diff
@@ -0,0 +1,83 @@
+licenses(["notice"])  # Apache 2
+
+load("//bazel:envoy_build_system.bzl", "envoy_package")
+load("@rules_foreign_cc//tools/build_defs:cmake.bzl", "cmake_external")
+
+envoy_package()
+
+cmake_external(
+    name = "ares",
+    cache_entries = {
+        "CARES_SHARED": "no",
+        "CARES_STATIC": "on",
+        "CMAKE_BUILD_TYPE": "RelWithDebInfo",
+    },
+    cmake_options = ["-GNinja"],
+    lib_source = "@com_github_c_ares_c_ares//:all",
+    make_commands = [
+        "ninja",
+        "ninja install",
+    ],
+    static_libraries = ["libcares.a"],
+)
+
+cmake_external(
+    name = "event",
+    cache_entries = {
+        "EVENT__DISABLE_OPENSSL": "on",
+        "EVENT__DISABLE_REGRESS": "on",
+        "CMAKE_BUILD_TYPE": "RelWithDebInfo",
```

Comment:
Nit: libevent used to be built in `Release` mode, I'm not sure if this change is intentional or accidental here.